src/import-page/file-section/FileSection.jsx

```diff
@@ -30,21 +30,20 @@ const FileSection = ({ intl, courseId }) => {
         subtitle={fileName && intl.formatMessage(messages.fileChosen, { fileName })}
       />
       <Card.Section className="px-3 pt-2 pb-4">
-        {isShowedDropzone
-          && (
-            <Dropzone
-              onProcessUpload={
-                ({ fileData, requestConfig, handleError }) => dispatch(handleProcessUpload(
-                  courseId,
-                  fileData,
-                  requestConfig,
-                  handleError,
-                ))
-              }
-              accept={{ 'application/gzip': ['.tar.gz'] }}
-              data-testid="dropzone"
-            />
-          )}
+        {!viewOnly && isShowedDropzone && (
```

Comment:
Is it better to disable the field and have an alert rather than hiding the entire dropzone?

Reply:
We considered both options, but the the `dropzone` component from paragon didn't have a disable option. The only way we were able to simulate that effect was to place a `button` around the `dropzone` and disable the `button`. We ultimately decided we thought it was a cleaner solution to remove the `dropzone` than to overlay with the `button`. We are open to the `button` option (or any others you're aware of) if you think that is a better fit for the CMS.

Reply:
Since the dropzone cannot be disabled, can you replace it with an alert notifying the user that they have view only access and cannot import a new file?

Reply:
@hsinkoff did you add this alert to the dropzone on the import page?

Reply:
No, I added it on the import page instead of the dropzone. I used the value of `viewOnly` to determine if the dropzone should be shown or the alert message should be shown in its place.